### ISSUE TITLE
857

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#473]

**Comment by corialanus** [comment]
@CECSpecialistI  this is a new MARC field from 2023 so it isn't in the spreadsheet list. I suspect it is nearly identical to 856 just for archived resources.

As I'm still very new to working with the spreadsheet I'll see if I can make a local copy of 856 (which looks to be mapped by Cypress) and work on that and email it to you.

I'm looking for quick wins initially to build my familiarity and comfort level :)

**Comment by CECSpecialistI** [comment]
It is identical to 856, just lacking a few should-be-obsolete indicator values like dial-up and such. This one should be easy for that reason I think!

**Comment by corialanus** [comment]
I've also added a note to the review to see comments here.

Subfield 'q' has turned out to be problematic as initially it appears to be a combination of content and carrier types so I mapped it to Manifestation: not on manifestation to be safe. However, I then tripped over the element Manifestation: encoding format which may also work as a mapping. Thus a second set of eyes would be valuable.

**Comment by lake44me** [comment]
@coriolanus I've just started to review this... I noticed there are no transformation notes, and that you are mapping all instances of subfield b (Archiving agency) using "appellation of agent".

Here's what's problematic - in order to code the developers ultimately need to relate the data from the MARC field to the manifestation (or its Work) they have created for "the thing this MARC record describes".  

Then subject of a property is assumed to be that Work if it's a Work property, or that Manifestation if it's a manifestation property.  But the subject of your first line would have to be a minted Agent entity, and that's not specified, and then does that Agent have a relationship with the Work or Manifestation of the record, or a related Work or Manifestation.... ?  Or, to an item-level Collection or collection work related to this or a related manifestation, maybe?  

The second indicator gives a picture of the relationship of the 857 to the W/M the MARC describes, so I think the mappings need to be conditioned on that....

I won't have time today to get into suggesting changes for this but do you want to revise, or wait for that?  You might look at some other spreadsheets with complex relationships and see what we ddid for $5 (in Decisions index).  This one might need a diagram like that.

**Comment by corialanus** [comment]
@lake44me I'll take another look. This ended up being more complex than it appeared at first glance.

**Comment by lake44me** [comment]
@corialanus it is indeed complex, and I looked and can't find any other field, other than mappings for fields that take $5, that deal with relating collections and their institutions to certain data.  At least none that we didn't turn into "note on manifestation"...

**Comment by lake44me** [comment]
I agree, I have changed all the First Pass mappings to Phase II in the spreadsheet (there were a lot among the "not mapped" and Deletes - over 600 lines~!) and marked it as Done for Phase I.
